src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs

```diff
@@ -74,6 +75,7 @@ public void Dispose()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #19456")]
```

Comment:
This should be reverted.  It was addressed by https://github.com/dotnet/corefx/pull/19477.